### ISSUE TITLE
[QUESTION] question about FixedBatchNormalization layer

### ISSUE THREAD

**Issue by likebullet86** [yhenon/keras-frcnn#33]
in this repository,
resnet layer use custom FixedBatchNormalization layer instead of keras official batchnormalization layer.

why do you use FixedBatchNormalization layer?
 - there are some bug in official batchnormalization layer?

**Comment by yhenon** [comment]
I'm just following what is described in the original resnet paper:

> For the usage of BN layers, after pre-training, we com-
pute the BN statistics (means and variances) for each layer
on the ImageNet training set. Then the BN layers are fixed
during  fine-tuning  for  object  detection.   As  such,  the  BN
layers become linear activations with constant offsets and
scales, and BN statistics are not updated by fine-tuning. We
fix the BN layers mainly for reducing memory consumption
in Faster R-CNN training.

**Comment by liuwei16** [comment]
@yhenon BN is fixed in the caffe implementation of Resnet50 for faster rcnn? or just for reducing memory?
When applying in our detection task, it is easy to overfit, I wonder whether it comes from the Fixed BN layer.

**Comment by mrgloom** [comment]
Can you point out where it's initialized with statistics from pretrained network?
https://github.com/yhenon/keras-frcnn/blob/0c5caf02550e92e333409685ae9e4a3afc4104aa/keras_frcnn/FixedBatchNormalization.py
